<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMainWindow</class>
 <widget class="QMainWindow" name="FMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_S"/>
    <addaction name="actionSave_as_A"/>
    <addaction name="separator"/>
    <addaction name="actionQuit_Q"/>
   </widget>
   <widget class="QMenu" name="menuCryptor_C">
    <property name="title">
     <string>Cryptor(&amp;C)</string>
    </property>
    <addaction name="actionEncryptor_E"/>
    <addaction name="actionDecryptor_D"/>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuCryptor_C"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_O">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_S">
   <property name="text">
    <string>Save(&amp;S)</string>
   </property>
  </action>
  <action name="actionSave_as_A">
   <property name="text">
    <string>Save as(&amp;A)</string>
   </property>
  </action>
  <action name="actionQuit_Q">
   <property name="text">
    <string>Quit(&amp;Q)</string>
   </property>
  </action>
  <action name="actionEncryptor_E">
   <property name="text">
    <string>Encryptor(&amp;E)</string>
   </property>
  </action>
  <action name="actionDecryptor_D">
   <property name="text">
    <string>Decryptor(&amp;D)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
